makei

```diff
@@ -188,13 +192,22 @@ main() {
 
     echo -e "${BLUE}makei: Reading from iproj.json${NOCOLOR}"
     objlib=$(parse_placeholder $(retrieve_from_iproj objlib "*CURLIB"))
+    curlib=$(parse_placeholder $(retrieve_from_iproj curlib "*CURLIB"))
     tgtCcsid=$(retrieve_from_iproj tgtccsid "*JOB")
     IBMiEnvCmdList=$(retrieve_from_iproj setIBMiEnvCmd[] "")
     preUsrliblRaw=$(retrieve_from_iproj preUsrlibl[] "")
     postUsrliblRaw=$(retrieve_from_iproj postUsrlibl[] "")
     preUsrlibl=""
     postUsrlibl=""
 
+    if [[ $curlib == "*CRTDFT" ]]; then
+        curlib="QGPL"
```

Comment:
I don't think we should do this, because QGPL does not get added to the library list as the current library
Can you just do a chgcurlib to *CURLIB